### DISCUSSION TITLE
Password confirmation dialog shows wrong branch

### DISCUSSION THREAD

**Discussion by twiddler** [planetscale/discussion#625] [Bug Reports] [answered]
When generating a new password at https://app.planetscale.com/foo/bar/settings/passwords, the confirmation dialog does not match the branch that was selected when creating the password. (The password works with the selected branch `dev`.)

![image](https://github.com/planetscale/discussion/assets/13364827/476c8844-31cb-4a3d-b250-868fb4366096)

![image](https://github.com/planetscale/discussion/assets/13364827/cad5f62a-93ce-4e89-99c7-21f1e86ffa1c)

**Comment by mscoutermarsh** [comment, marked as answer]
@twiddler Thank you! Got it logged. We'll get this fixed.

**Comment by mscoutermarsh** [comment]
Update: fix is deploying now.
